src/main/java/pl/edu/zut/mad/schedule/login/LoginActivity.kt

```diff
@@ -69,4 +56,25 @@ internal class LoginActivity : AppCompatActivity(), LoginMvp.View {
         setResult(Activity.RESULT_OK)
         finish()
     }
+
+    override fun getActivityComponent() = component
+
+    private fun initComponent() {
```

Comment:
Not sure if you need these two methods for better readability. Code itself is probably descriptive enough.

Reply:
This is remainings of old solution with mocking DI, I'll remove them.